src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder salaryInfo = new StringBuilder();
+        LocalDate startDate = LocalDate.parse(dateFrom);
+        LocalDate endDate = LocalDate.parse(dateTo);
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy");
+        salaryInfo.append("Report for period ");
+        salaryInfo.append(dateFrom);
+        salaryInfo.append(" - ");
+        salaryInfo.append(dateTo);
+        for (String name : names) {
+            int salary = 0;
+            for (String datum : data) {
+                String[] lines = datum.split(" ");
+                LocalDate date = LocalDate.parse(lines[0]);
```

Comment:
Similar to the previous comment, ensure that the date format in the data matches the expected input format. Use the DateTimeFormatter defined on line 11 if necessary.